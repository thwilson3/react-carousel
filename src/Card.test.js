import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon.js";

it("renders without crashing", function () {
  render(
    <Card
      caption={TEST_IMAGES[0].caption}
      src={TEST_IMAGES[0].src}
      currNum={2}
      totalNum={75}
    />
  );
});

it("matches snapshot", function () {
  const { container } = render(
    <Card
      caption={TEST_IMAGES[0].caption}
      src={TEST_IMAGES[0].src}
      currNum={2}
      totalNum={75}
    />
  );
  expect(container).toMatchSnapshot();

});

// Card({caption, src, currNum, totalNum})
