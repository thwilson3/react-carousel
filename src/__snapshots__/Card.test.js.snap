// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      testing image 1
    </h4>
    <img
      alt="testing image 1"
      class="Card-image"
      src="test1.com"
    />
    <small
      class="Card-small"
    >
      Image 
      2
       of 
      75
      .
    </small>
  </div>
</div>
`;
